### ISSUE TITLE
failed to install vddos nginx on centos 7

### ISSUE THREAD

**Issue by centminmod** [duy13/vDDoS-Protection#22]
From https://community.centminmod.com/threads/ddos-protection.14431/page-2#post-62015

After install

```
/etc/init.d/vdos
Usage: /etc/init.d/vdos {start|stop|status|restart|condrestart|try-restart|reload|force-reload|configtest}

/etc/init.d/vdos status
Unit vdos.service could not be found.

ls -lah /etc/init.d/vdos
-rwx------ 1 root root 1.6K Apr  2 03:30 /etc/init.d/vdos
```

vddos nginx config check, it's missing /vddos/vddos.conf which is i assume the nginx.conf equivalent. From auditd logging somewhere in /usr/bin/vddos setup routine it unlink removed /vddos/vddos.conf for some reason ?

```
ausearch -k vddos | aureport -f -i | fgrep 'vddos.conf'
32. 04/07/2018 12:56:50 /vddos/vddos.conf open yes /usr/bin/cp root 2987
44. 04/07/2018 12:57:06 /vddos/vddos.conf unlinkat yes /usr/bin/rm root 3014
```

```
/vddos/vddos -t
nginx: [emerg] open() "/vddos/vddos.conf" failed (2: No such file or directory)
nginx: configuration file /vddos/vddos.conf test failed
```

vdos nginx module directory missing

```
ls -lah /usr/lib64/vddos/modules
ls: cannot access /usr/lib64/vddos/modules: No such file or directory
```

**Comment by duy13** [comment]
Yes, everything you mention is in my arrangements. For example: "/etc/init.d/vdos" is only used when needed to check for errors during use, vdos is not used by the user. And "vddos.conf" was removed to hide the configuration (because I did not want to expose this part) in the attacker's challenge.
So, I wonder what you are looking for? I think it would be inefficient to find a way for vDDoS to run with Centminmod so I wrote a small bash script to deploy the same challenge filter on Centminmod:

https://community.centminmod.com/threads/anti-ddos-mitigate-automatic-deploy-config-testcookie-module-for-centminmod-vcentminmod-ddos.13258/

**Comment by centminmod** [comment]
oh i thought vddos nginx is a running process /service ? any documentation to read as to what vddos's working components are ?

yes i remember that link :)

**Comment by centminmod** [comment]
Cheers thanks for heads up - only tried vddos as one of my centmin mod users said they will install vddos on centminmod.
